sdk/ml/azure-ai-ml/azure/ai/ml/operations/_workspace_operations.py

```diff
@@ -142,7 +142,6 @@ def begin_sync_keys(self, name: str = None) -> LROPoller:
     def begin_create(
         self,
         workspace: Workspace,
-        update_dependent_resources: bool = False,
```

Comment:
Is this expected to be removed? This will count as breaking change

Reply:
> Is this expected to be removed? This will count as breaking change

thanks for the suggestion, reverted it.